Runtime/CourseController/AdvancedCourseController.cs

```diff
@@ -97,6 +98,18 @@ public class AdvancedCourseController : MonoBehaviour
 
         private void Awake()
         {
+            try
+            {
+                // Load training course from a file.
+                string coursePath = RuntimeConfigurator.Instance.GetSelectedCourse();
+                trainingCourse = RuntimeConfigurator.Configuration.LoadCourse(coursePath);
+            }
+            catch (Exception exception)
+            {
+                Debug.LogError($"{exception.GetType().Name}, {exception.Message}\n{exception.StackTrace}", RuntimeConfigurator.Instance.gameObject);
+                return;
+            }
+            
```

Comment:
There was a reason why we moved this after loading the localization files. See here and commit message:
https://github.com/Innoactive/Creator-Base-Template/commit/fa7f7da9df709aea7f05f356fbb6a7b8eb3d4646

Reply:
Is this still valid? @Maxsiom @Gusinuhe

Reply:
Nope
It's fixed.